src/register/RegistrationPage.jsx

```diff
@@ -85,7 +85,7 @@ class RegistrationPage extends React.Component {
       failureCount: 0,
       startTime: Date.now(),
       totalRegistrationTime: 0,
-      optimizelyExperimentName: '',
+      optimizelyExperimentName: '', // eslint-disable-line react/no-unused-state
```

Comment:
Can we remove this state as well alongside `getExperiments()`? I don't think we have any new experiments to launch now.

Reply:
okay